src/docs/common/snippets/common-graalpy-graalvm.adoc

```diff
@@ -0,0 +1,71 @@
+:exclude-for-languages:groovy
+
+== Python Runtime Compilation
+
+With GraalVM JDK, the Python code executed using the GraalPy engine will be just-in-time compiled.
+With other JDKs, the Python code will be interpreted, which negatively impacts its performance.
```

Comment:
```suggestion
With other JDKs, the Python code is interpreted, resulting in reduced performance.
```
I replaced "which negatively impacts its performance" with "resulting in reduced performance"